apps/email-signature-generator/src/index.css

```
@@ -0,0 +1,16 @@
+*, *::before, *::after { box-sizing: border-box; margin: 0; padding: 0; }
+
+body {
+  background: #080808;
+  color: #f0f0f0;
+  font-family: 'Outfit', 'Inter', system-ui, sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)